### DISCUSSION TITLE
LSF clusterOptions -m does not submit to specific server

### DISCUSSION THREAD

**Discussion by colindaven** [nextflow-io/nextflow#3359] [Q&A] [answered]
## Bug report 

LSF clusterOptions -m does not submit to specific server
Note  "" should be present according to LSF docs
```
executor {
  name = 'lsf'
  //name = 'local'
  perJobMemLimit = true
  queue = 'default'
  memory = '60 GB'
  cpus = 16
  clusterOptions = '-m \"myserver\"'
}

// others tried without success
  clusterOptions = '-m "myserver" '
  clusterOptions = '-m myserver'
```

### Expected behavior and actual behavior

Expected - job should only be submitted to specific server. 
Actual - jobs land on servers at random

### Steps to reproduce the problem

LSF cluster and changes in nextflow config only required 

### Environment 

* Nextflow version: 22.04 and 22.10
* Java version: 11.0.1
* Operating system: Linux
* Bash version: GNU bash, version 4.4.20(1)-release (x86_64-pc-linux-gnu)

### Additional context

Thanks

**Comment by jemunro** [comment, marked as answer]
clusterOptions is a process directive so doesn't belong in the executor scope.

Instead in your nextflow.config try:
```nextflow
process {
    clusterOptions = { task.executor == 'lsf' ? '-m "myserver"' : null }
}

executor {
  name = 'lsf'
  //name = 'local'
  perJobMemLimit = true
  queue = 'default'
  memory = '60 GB'
  cpus = 16
}
```

**Reply by colindaven** [reply]
Thanks, missed that in the docs somehow since it is all hidden in the Executor section. 

For those coming later, use spaces to separate multiple servers:

```
process {
    clusterOptions = { task.executor == 'lsf' ? '-m "server1 server2"' : null }
}
```
